.github/workflows/containabot.yml

```diff
@@ -0,0 +1,57 @@
+# Regularly updates the CI container
+name: Containabot
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 13 * * 1'
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    environment: main
+    steps:
+      - name: Checkout NVIDIA/Megatron-LM
+        uses: actions/checkout@v4
+        with:
+          repository: NVIDIA/Megatron-LM
+          ref: main
+          path: ${{ github.run_id }}
+
+      - name: Get latest mcore commit
+        id: ref
+        run: |
+          cd ${{ github.run_id }}      
+          sha=$(git rev-parse origin/main)
+          echo "sha=${sha}" >> "$GITHUB_OUTPUT"
+          echo "short_sha=${sha:0:7}" >> "$GITHUB_OUTPUT"
+
+      - name: Checkout ${{ github.repository }}
+        uses: actions/checkout@v4
+        with:
+          path: ${{ github.run_id }}
+          token: ${{ secrets.PAT }}
+
+      - name: Bump MCORE_TAG
+        run: |
+          cd ${{ github.run_id }}     
+          sed -i 's/^ARG MCORE_TAG=.*$/ARG MCORE_TAG=${{ steps.ref.outputs.sha }}/' Dockerfile.ci
+
+      - name: Create Bump PR
+        uses: peter-evans/create-pull-request@v6
+        id: create-pull-request
+        with:
+          path: ${{ github.run_id }}
+          branch: bump-ci-container
```

Comment:
maybe we can add date to the name of the branch to not have collisions
Collisions would be problematic if at day n it passes but at day n + 1 it overwrites (due for force push) but starts failing